web-main/src/app/state/core/user/user.effects.ts

```diff
@@ -72,6 +75,17 @@ export class UserEffects {
   ))
 
   // TODO: Implement onApiListOnlineIdsStarted$ => similar to onApiListStarted$
+  
+  onApiListOnlineIdsStarted$ = ngrxEffects.createEffect(() => this._actions.pipe(
+    ngrxEffects.ofType(User.Api.ListOnlineIds.actions.started),
+    rxjs.switchMap(() => this._userApiService.listOnlineIds().pipe(
+      rxjs.map(users => new Set(users.map(user => user.id))),
```

Comment:
Two things, you should never have two maps, one after another, because why don't do it in one? - right?

Second think, you shouldn't be passing 'new set' to a `succeeded` command, you can pass an array(list) and if the payload of the `succeeded` action doesn't like that we should change typing of that paylod.